presidio-image-redactor/presidio_image_redactor/image_analyzer_engine.py

```diff
@@ -23,15 +27,26 @@ def __init__(self, analyzer_engine: AnalyzerEngine = None, ocr: OCR = None):
             ocr = TesseractOCR()
         self.ocr = ocr
 
-    def analyze(self, image: object, **kwargs) -> List[ImageRecognizerResult]:
+    def analyze(
+        self, image: object, ocr_kwargs: Optional[dict] = None, **kwargs
```

Comment:
consider renaming `kwargs` to `text_kwargs`. This would be backward compatible as we call it with **.

Reply:
Agreed, that does make things clearer. I've added another commit renaming `kwargs` to `text_analyzer_kwargs` in `ImageAnalyzerEngine` and all the classes that pass kwargs to it.